src/app/shared/notification/service/notification.service.ts

```diff
@@ -53,10 +55,26 @@ export class NotificationService {
   }
 
   public rateComplexNavigator(): void {
-    this.addRatingNotification('<div><input class="input-small" value="textbox"/><a href="' + environment.complex_portal_support_url + '">Tell us about your experience with the Complex Navigator!</a></div>');
+    const ratingHtml = `
+  <div class="ratingComponent">
+    <div class="ratingComponentStars">
+      <span class="star" [ngClass]="{'selected': selectedRating >= 1}" (click)="selectRating(1)"> &#9733; </span>
+      <span class="star" [ngClass]="{'selected': selectedRating >= 2}" (click)="selectRating(2)"> &#9733; </span>
+      <span class="star" [ngClass]="{'selected': selectedRating >= 3}" (click)="selectRating(3)"> &#9733; </span>
+      <span class="star" [ngClass]="{'selected': selectedRating >= 4}" (click)="selectRating(4)"> &#9733; </span>
+      <span class="star" [ngClass]="{'selected': selectedRating >= 5}" (click)="selectRating(5)"> &#9733; </span>
+    </div>
+    <input type="submit" name="submitButton" value="Submit Rating" class="button" (click)="submitRating()">
+  </div>
+    `;
+    this.addRatingNotification(ratingHtml);
```

Comment:
I've tried but I haven't managed to make the buttons inside the toast clickable. I even used `toastComponent` and created a new component but no luck. I did a quick search and it may be that this type of toast messages don't work with clickable buttons in it.